tests/PHPUnit/Model/KataTest.php

```diff
@@ -43,4 +45,14 @@ public function test_start_should_return_started_kata()
         $this->kata->addStep($this->getMock(Step::INTERFACE_NAME));
         $this->assertInstanceOf(StartedKata::CLASS_NAME, $this->kata->start());
     }
+
+    public function test_should_contain_objectives()
+    {
+        $this->assertAttributeCount(0, 'objectives', $this->kata);
+        $this->kata->addObjective(new NullObjective());
+        $this->assertAttributeCount(1, 'objectives', $this->kata);
+        $this->kata->addObjective(new NullObjective());
+        $this->kata->addObjective(new NullObjective());
+        $this->assertAttributeCount(3, 'objectives', $this->kata);
```

Comment:
what benefit does this test add? You don't know that the class is using the 3 objectives properly.. just that it has them .. which isn't very useful to know at all.